akka-stream-testkit/src/main/scala/akka/stream/testkit/StreamTestKit.scala

```diff
@@ -185,23 +188,43 @@ object TestSubscriber {
 
     private val probe = TestProbe()
 
+    @volatile private var _subscription: Subscription = _
+
     private val self = this.asInstanceOf[Self]
 
     /**
-     * Expect and return a Subscription.
+     * Expect and return a [[Subscription]].
+     */
+    def expectSubscription(): Subscription = {
+      _subscription = probe.expectMsgType[OnSubscribe].subscription
+      _subscription
+    }
+
+    /**
+     * Expect and return [[SubscriberEvent]] (any of: `OnSubscribe`, `OnNext`, `OnError` or `OnComplete`).
      */
-    def expectSubscription(): Subscription = probe.expectMsgType[OnSubscribe].subscription
+    def expectEvent(): SubscriberEvent =
+      probe.expectMsgType[SubscriberEvent]
 
     /**
-     * Expect [[SubscriberEvent]].
+     * Fluent DSL
+     *
+     * Expect [[SubscriberEvent]] (any of: `OnSubscribe`, `OnNext`, `OnError` or `OnComplete`).
      */
-    def expectEvent(event: SubscriberEvent): Self = {
+    def expectEvent(event: SubscriberEvent): Self = { // TODO it's more "signal" than event, shall we rename? -- ktoso
```

Comment:
hmm, I think “event” is also okay, I don’t see it being confused with “stream element” (if that was your worry)